File: assets.js
function loadAssets() {
  const assets = {
      background: loadImage('resources/background.png'),
      sushi: loadImage('resources/sushi.png'),
      pipeTop: loadImage('resources/pipe_top.png'),
      pipeBottom: loadImage('resources/pipe_bottom.png'),
  };
  return assets;
}

function loadImage(src) {
  const img = new Image();
  img.src = src;
  return img;
}

File: FlappySushi_Python/assets.py
import pygame

def load_assets():
    assets = {
        "background": pygame.image.load("../resources/background.png"),
        "sushi": pygame.image.load("../resources/sushi.png"),
        "pipe_top": pygame.image.load("../resources/pipe_top.png"),
        "pipe_bottom": pygame.image.load("../resources/pipe_bottom.png"),
    }
    return assets
